lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -478,6 +498,15 @@ export class UserAgentApplication {
         return;
       }
 
+      // if extraScopesToConsent is passed, append them to the login request
+      let scopes: Array<string>;
+      if (request.extraScopesToConsent) {
+        let scopes = [...request.scopes, ...request.extraScopesToConsent];
```

Comment:
scopes has already been defined above, no need to have let to set it

Reply:
Changes done.